schema/sourcecode.md

```diff
@@ -1,87 +1,131 @@
 
-# Software Source Code
+# Software Source Code Metadata
 
-This document outlines the *required* and *recommended* properties for
-describing source code.
+To classify a record as a computer programming source code, `"@type: "SoftwareSourceCode"` should be used in the json schema. This will classify the record such as compile ready solutions, code snippet samples, scripts, etc. as a specific Schema.Org type called `SoftwareSourceCode` for which the metadata should be described using the [core metadata](https://github.com/I-GUIDE/data-catalog/blob/main/schema/core.md), as well as the software-source-code-specific properties for the [Schema:SoftwareSourceCode](https://schema.org/SoftwareSourceCode) class. The following table outlines the required and optional properties selected from Schema.Org vocabulary to design the I-GUIDE software source code metadata schema. These properties are encoded as `1` or `1+` for **required** and `0,1` or `0+` for **optional** in the Cardinality column of the table below.
```

Comment:
Consider changing to something like the following:

To classify a record as a computer programming source code, use `"@type: "SoftwareSourceCode"` in the json schema. This is appropriate for records including code snippet samples, scripts, notebooks, etc. The following table outlines the required and optional properties to sufficiently describe software source code objects. **Required** properties have a cardinality of `1` or `1+` and **optional** properties have a cardinalities of `0`, `0+`, `1`.